azure/scope/managedcontrolplane.go

```diff
@@ -51,12 +48,15 @@ import (
 // control plane.
 type ManagedControlPlaneScopeParams struct {
 	AzureClients
-	Client           client.Client
-	Cluster          *clusterv1.Cluster
-	ControlPlane     *infrav1exp.AzureManagedControlPlane
+	Client              client.Client
+	Cluster             *clusterv1.Cluster
+	ControlPlane        *infrav1exp.AzureManagedControlPlane
+	ManagedMachinePools []ManagedMachinePool
+}
+
+type ManagedMachinePool struct {
```

Comment:
Can we move this to `managedmachinepool.go` since we're trying to separate the scopes? Or would that cause an import cycle?